src/config/experimental.ts

```diff
@@ -22,6 +23,7 @@ export const defaultConfig = {
   [NOTIFICATIONS]: { needsRestart: true, settings: true, value: false },
   [PROFILES]: { settings: true, value: true },
   [REVIEW_ANDROID]: { settings: false, value: false },
+  [CROSSCHAIN_SWAPS]: { settings: true, value: false },
```

Comment:
i think having just this is perfect for now, then when we start adding new stuff on top we can use the flag more broadly